browser/sources.gni

```diff
@@ -609,6 +609,12 @@ brave_chrome_browser_allow_circular_includes_from = [
   "//brave/browser/brave_stats",
 ]
 
+if (toolkit_views) {
+  brave_chrome_browser_deps += [ "//brave/browser/ui/sidebar" ]
+  brave_chrome_browser_allow_circular_includes_from +=
```

Comment:
I missed that you're using `brave_chrome_browser_allow_circular_includes_from` instead of `brave_ui_allow_circular_includes_from`. This is a problem we need to fix as we should not be adding new circular dependencies. This needs a follow-up to fix.